docs/source/start_all.rst

```diff
@@ -35,13 +35,13 @@ If not used, each run of the containers will clear all of Grafana information.
 
 **-j dashboard** Allows adding dashboards to Grafana, multiple parameters are supported.
 
-**-c grafana environment variable** Grafana allows overridden its configuration from the environment. The ``-c`` flag, allows adding an environment variable to Grafana and altering its configuration by doing so.
```

Comment:
The meaning of what needs to be written here is  that you can override Grafana's configuration using the environment (specifically environment variables)